pypeit/spectrographs/gemini_gmos.py

```diff
@@ -145,14 +154,20 @@ def compound_meta(self, headarr, meta_key):
             # binning in the raw frames
             ccdsum = headarr[1].get('CCDSUM')
             if ccdsum is not None:
-                binspatial, binspec = parse.parse_binning(ccdsum)
+                binspec, binspatial = parse.parse_binning(ccdsum)
                 binning = parse.binning2string(binspec, binspatial)
             else:
                 # binning in the spec2d file
                 binning = headarr[0].get('BINNING')
             if binning is None:
                 msgs.error('Binning not found')
             return binning
+        elif meta_key == 'mjd':
+            obsepoch = headarr[0].get('OBSEPOCH')
+            if obsepoch is not None:
+                return time.Time(obsepoch, format='jyear').mjd
+            else:
+                return None
```

Comment:
you should update the docstring as well, because you are return None instead of a Metadata value read from the header(s)

Reply:
Agreed. I also wonder if it might be worthwhile throwing an error, or, throw a warning and then assume that the observations have ben taken since "2023-12-14".

Reply:
Indeed, and I agree, but returning None could cause an error on line 846 (and several other places in this file where a possible None value is not checked)... I think?

Reply:
Ah, yes, you're right.  @debora-pe , you'll need to catch the case @rcooke-ast pointed out.  There are other places in the code that can catch a None mjd (e.g., starting at line 255 in `metadata.py`), but maybe not everywhere.